### DISCUSSION TITLE
How much VRAM is needed for finetuning the large model

### DISCUSSION THREAD

**Discussion by dgoryeo** [openai/whisper#1666] [Q&A] [answered]
Would finetuning Whisper require significantlt more VRAM? 
I am planning for finetuning the large-v2 model. I have two use case scenraios:

- (a) Finetuning for additional vocabolary of say 200,000 words. 
- (b) Finetuning for additional language dataset

Any pointers or rough range would be helpful.

**Comment by glangford** [comment, marked as answer]
fyi, see

Memory requirements for local training
https://huggingface.co/openai/whisper-large-v2/discussions/21

**Comment by dgoryeo** [comment]
Thanks @glangford !!!
